doc/classes/TextServer.xml

```diff
@@ -391,11 +399,11 @@
 				Returns [Dictionary] with OpenType font name strings (localized font names, version, description, license information, sample text, etc.).
 			</description>
 		</method>
-		<method name="font_get_oversampling" qualifiers="const">
+		<method name="font_get_oversampling" qualifiers="const" deprecated="Use [Viewport] oversampling, or [code skip-lint]oversampling[/code] argument of the [code skip-lint]draw_*[/code] methods instead.">
 			<return type="float" />
 			<param index="0" name="font_rid" type="RID" />
 			<description>
-				Returns font oversampling factor, if set to [code]0.0[/code] global oversampling factor is used instead. Used by dynamic fonts only.
+				Deprecated. This method do nothing.
```

Comment:
It does return something.

Reply:
Changed it to `This method always returns [code]1.0[/code].` to be more precise.